tests/ExpectMutateInit.json

```diff
@@ -198,7 +191,6 @@
         "admission.trusted.identity/status": "injected",
         "admission.trusted.identity/ti-cluster-name": "testCluster",
         "admission.trusted.identity/ti-cluster-region": "testRegion",
-        "admission.trusted.identity/ti-images": "trustedseriviceidentity/myubuntu:latest@sha256:5b224e11f0e8daf35deb9aebc86218f1c444d2b88f89c57420a61b1b3c24584c",
-        "admission.trusted.identity/ti-secret-key": "ti-secret-XXX"
+        "admission.trusted.identity/ti-images": "trustedseriviceidentity/myubuntu:latest@sha256:5b224e11f0e8daf35deb9aebc86218f1c444d2b88f89c57420a61b1b3c24584c"
```

Comment:
I think we need to update this to image formats that are supported by openshift as well.

Since tag + sha256 isn't supported, should just be `img@sha256:...` without tag.

Reply:
Done